api/src/opentrons/protocol_api/protocol_context.py

```diff
@@ -423,7 +424,8 @@ def move_labware(
                         using :py:meth:`load_labware`
 
         :param new_location: Deck slot location or a hardware module that is already
-                             loaded on the deck using :py:meth:`load_module`.
+                             loaded on the deck using :py:meth:`load_module` or off deck using :py:type OFF_DECK.
+
```

Comment:
reStructuredText typo:

```suggestion
                             loaded on the deck using :py:meth:`load_module`
                             or off deck using :py:obj:`OFF_DECK`.
```

Note that this will cause a new warning to be raised in the docs build, because `OFF_DECK` isn't listed in the reference documentation, so Sphinx doesn't know how to link to it. We'll need to fix that at some point, so people know how to import `opentrons.protocol_api.OFF_DECK`. But that doesn't have to happen in this PR.